plugin/Installer/DatabaseChanges.php

```diff
@@ -44,7 +44,7 @@
 class DatabaseChanges implements DatabaseChangesInterface
 {
 	/** @var int */
-	const MAX_VERSION = 38;
+	const MAX_VERSION = 39;
```

Comment:
@yeneastgate see the comment above for suggestions please :)

Reply:
@andernath I suggest fixing the current branch (32990 prices on request) and merging it to the master branch first.
Then I will check and fix the conflicts in the branches: 34563 Map in property listing & 32986 Default values ​​for estate lists.

Reply:
@andernath @fredericalpers Yes, I will check again and confirm with you later

Reply:
@andernath After checking again, I suggest that:
1. Based on the current code, we will merge the branch "32990-price-on-request"  first. Then merge the branch "34563-map-in-property-listing"  and the branch "32986-default-values-for-estate-lists" at the end.
2. Or in case if you want to release any features first. Please, let us know and we will fix the conflict of that feature first.